src/referrals/referrals.service.ts

```
@@ -90,6 +91,14 @@ export class ReferralsService {
     }
   }
 
+  async getReferralById(id: string) {
+    const entity = await this.referralRepo.findOne({ where: { id } });
+    if (!entity) {
+      throw new NotFoundException(`Referral with id '${id}' not found`);
+    }
+    return this.normalizeReferral({ ...entity, referLink: buildReferLink(entity.slug) });
```

ESLint

[error] Replace `·...entity,·referLink:·buildReferLink(entity.slug)` with `⏎······...entity,⏎······referLink:·buildReferLink(entity.slug),⏎···`

(prettier/prettier)